ansible/roles/ocp4-workload-cnvlab/meta/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+galaxy_info:
+  role_name: ocp4-workload-cnvlab
+  author: Red Hat Field Product Management
```

Comment:
Would you mind adding an e-mail address or other contact info, please? The Ops team needs a way to find folks to call if (when?) this breaks.